packages/service/index.d.ts

```diff
@@ -12,6 +13,15 @@ import { ServiceGenerator } from './lib/generator/service-generator'
 /* eslint-disable @typescript-eslint/no-unused-vars */
 export import Generator = ServiceGenerator.ServiceGenerator
 
+export class Bus extends EventEmitter {
+  constructor (id: string)
+
+  close: () => void
+
+  send: (destination: string, type: string, data: any) => void
+  broadcast: (destination: string, type: string, data: any) => void
+}
```

Comment:
please add a test for this type. We use `tsd`

Reply:
Done!